teacher.rb

```diff
@@ -0,0 +1,26 @@
+require './main'
+# Public Teacher class used for creating teacher objects
+#  Teacher class inherits from person class.
+#
+# All methods are class methods and should be called on the Person class.
+#
+# Examples
+#
+# teacher = Teacher.new('unknown')
+# teacher.can_use_services?
+# # => true
+class Teacher < Person
+  attr_reader :specialization
```

Comment:
- [ ] This `attr_reader` is not needed since we are creating it in the constructor.